Update getting started Wi-Fi section (#14187)
home-assistant/home-assistant.github.io: source/getting-started/index.markdown

1. Put the SD card in your card reader.
2. Open balenaEtcher, select the Home Assistant image and flash it to the SD card.
3. Unmount the SD card and remove it from your card reader.

> 4. The most foolproof networking setup for your Raspberry Pi is to connect it using an Ethernet cable, however if you want to configure Wi-Fi or a static IP address (this step requires a USB stick) you can try this: (Note: Your milage may vary and Wi-Fi might not work even after taking these steps.)

Reviewer: Let's not call it `foolproof`, as it can be condescending towards people who run into issues.

```suggestion
4. The most reliable networking setup for your Raspberry Pi is to connect it using an Ethernet cable, however, if you want to configure Wi-Fi or a static IP address (this step requires a USB stick) you can try this:
```

Let's remove the last note as well. Wi-Fi should work and is supported. If it doesn't, a bug should be raised.

Author: Very well. WiFi didn't work for me at all though and I tried everything. I have seen others in the forum's having the same problems. But I'll raise a bug then.

Reviewer: 👍  Opening a bug report and getting stuff fixed is always the best way forward.

Nevertheless, I do think we can merge in your changes if the above suggestion is applied.
